cmake/install.cmake

```diff
@@ -87,22 +90,35 @@ if(NOT MSVC)
 else()
   set(CMAKE_INSTALL_CMAKEDIR "cmake" CACHE STRING "${_cmakedir_desc}")
 endif()
+mark_as_advanced(CMAKE_INSTALL_CMAKEDIR)
```

Comment:
For example CMAKE_INSTALL_BINDIR and other is advanced variables.
They are used by advanced users and should not interfere with the basic process.
Variable CMAKE_INSTALL_PREFIX know and use all - it does not need to hide in the normal mode.